python-manual/modules/ROOT/pages/transactions.adoc

```diff
@@ -0,0 +1,289 @@
+= Fine-grained transaction control
```

Comment:
Indeed. This is my solution to the void generated by taking away `execute_query`. Without `execute_query`, there is no way to make queries without sessions and transactions and `Result` methods. Which is a lot to internalize before running even one query. So my solution is that, in the basic workflow, we pretend that managed transactions are just a code structure (almost boilerplate code) that you have to use to run queries, with not much justification. But at least it gets you started. Then we can revise the sloppy, black-box attitude in the subsequent pages. But I think that if I were to start explaining all the content of `transactions.adoc` before being able to do `MATCH (n:Person)`, the learning curve would just be too discouraging. So I have to skip over a lot of details. But I also want both pages to be somewhat self-contained, so that you can land on either and not be required to read the other. I'll try to do some more work to cross-reference them, but I hope this explains the rationale behind it at least :)

Reply:
That makes perfect sense. :ok_hand: People reading them in sequence will just need to skim over some parts they already know about. Better than leaving others puzzled, probably.